app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,58 @@
+module Api
+  module V1
+      class UsersController < ApplicationController
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/blyscuit/rails-example-one/blob/ffe479ef97670c7333f37452d55369f226bbc764/app/controllers/api/v1/users_controller.rb#L3"></span>Use 2 (not 4) spaces for indentation.</td>
    </tr>
  </tbody>
</table>